### PR TITLE
#338 - Tooltip for Report button

### PR DESCRIPTION
**Pull request by Klotske**
Worked on #338

As requested by comment https://github.com/LAION-AI/Open-Assistant/issues/338#issuecomment-1370184588 added tooltip for the report buttons

**Before:**
Before
**After:**
After

### CODE REVIEW

**Comment by andrewm4894** [comment]
cool, thanks, might even be able to use this to try maybe add some myself elsewhere perhaps. 

one thing is what does "Report" mean? is it to report or flag something as sensitive or inappropriate, or broken in some sense? 

just asking as im actually not sure so wondering if something we can say thats more specific without getting too wordy.

File: website/src/components/FlaggableElement.tsx [hunk 1]
```diff
@@ -69,11 +70,15 @@ export const FlaggableElement = (props) => {
     >
       <Grid templateColumns="1fr min-content" gap={2}>
         <PopoverAnchor>{props.children}</PopoverAnchor>
-        <PopoverTrigger>
-          <Button h="full">
-            <FlagIcon className="w-4 text-gray-400 group-hover:text-gray-500" aria-hidden="true" />
-          </Button>
-        </PopoverTrigger>
+        <Tooltip hasArrow label="Report" bg="red.600">
+          <div>
```
Comment (AbdBarho) [inline website/src/components/FlaggableElement.tsx:74]: is this div necessary?

Comment (Klotske) [reply website/src/components/FlaggableElement.tsx:74]: Yeah. The Tooltip component refuses to work if its child element is a PopoverTrigger. Same thing in reverse. Tried several approaches but only this one worked.

Comment (fozziethebeat) [reply website/src/components/FlaggableElement.tsx:74]: I concur, the Chakra docs show using `Box` which is basically `div`.

**Review by fozziethebeat — APPROVED** [review]
